app/models/katello/product.rb

```diff
@@ -258,5 +258,9 @@ def self.unused_product_id
         id
       end
     end
+
+    def self.all_products(organization_id)
+      ::Katello::Product.joins(:subscriptions => :pools).where(:organization_id => organization_id).enabled.uniq
+    end
```

Comment:
This is the last open concern other than missing the issue # in the commit and subject message